apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -287,6 +329,7 @@ export default NextAuth({
           username: user.username,
           email: user.email,
           role: user.role,
+          sessionTimeout: user?.sessionTimeout,
```

Comment:
Making sure we have the `sessionTimeout` value to use it when encoding the session token